pkg/resources/manager/manager.go

```diff
@@ -530,3 +572,21 @@ func removeSeparatorFromEnd(in string) string {
 
 	return in[:len(in)-len(sep)]
 }
+
+func loadMachineClasses(reader io.Reader) ([]*MachineClass, error) {
```

Comment:
i don't see big benefits in MCR anymore and i kept it for compatibility with upstream. I moved here because MachineClass is type under resource manager so i wanted keep it together.